mantidproject/mantid
# IndirectDiffraction - Post Plot & Save

The plotting and saving of the results workspace has been changed to after the execution of the algorithm for the Indirect Diffraction interface.

**To test:**

Check each plot option plots a suitable graph and that the appropriate workspaces have been saved in your save directory.
Test for instruments IRIS and OSIRIS
\olympic\babylon5\Public\Louise McCann\Diffraction

Fixes #17024.



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
